src/server/game/Miscellaneous/Formulas.cpp

```diff
@@ -93,7 +93,37 @@ uint32 Acore::XP::Gain(Player* player, Unit* unit, bool isBattleGround /*= false
             xpMod *= creature->GetCreatureTemplate()->ModExperience;
         }
 
-        xpMod *= isBattleGround ? sWorld->getRate(RATE_XP_BG_KILL) : sWorld->getRate(RATE_XP_KILL);
+        if (isBattleGround)
+        {
+            switch (player->GetMapId())
+            {
+                case 30:  // AV
```

Comment:
isn't there an enum mapping all these IDs? if not, shall we create one?

Reply:
Sadly no, there is no enum :( I would but I don't know where exactly, hehe

Reply:
map.h or something

Reply:
ping ping :)

Reply:
Done, I added it on Battleground.h since it makes more sense than Map.h, more specific...